src/generated/cloud/redis/cluster/v1/src/model.rs

```diff
@@ -3711,6 +3715,7 @@ pub mod connection_detail {
     /// Connectivity Automation (auto-registered connection) during the cluster
     /// creation, or it could be created by customer themselves (user-registered
     /// connection).
+    #[serde_with::serde_as]
```

Comment:
It is a bit sad that we have all these annotations that mostly go unused. They are also harmless, so LGTM